src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -90,10 +91,12 @@ case class StateManager(depositDir: File, submitBase: File, easyHome: URL) exten
         // probably properly saved without toString but getSubmittedBagId would throw
         // ConversionException: 'bag-store.bag-id' doesn't map to a String object
         draftProps.setProperty(bagIdKey, bagStoreBagId.toString)
+        draftProps.setProperty(depositSourceKey, "API")
```

Comment:
Following the comment in https://github.com/DANS-KNAW/easy-specs/pull/82#discussion_r310344792, please rename this property to `deposit.origin`. Also rename the `depositSourceKey` value to `depositOriginKey`.